apps/src/templates/sectionProgress/SectionProgress.jsx

```diff
@@ -107,10 +103,12 @@ class SectionProgress extends Component {
               scriptId={scriptId}
               onChange={this.onChangeScript}
             />
-            <LessonSelector
-              lessonNumbers={lessonNumbers}
-              onChange={this.onChangeLevel}
-            />
+            {lessons.length !== 0 &&
```

Comment:
do we have no lessons only when we're loading? I wonder if it'd be useful to set up a "loading" variable that gets used throughout render (in a later PR) since it seems like we might have to deal with this a bunch.